packages/dynamite/dynamite_runtime/lib/src/client/response.dart

```diff
@@ -1,9 +1,10 @@
 import 'dart:convert';
-import 'dart:typed_data';
+import 'dart:developer';
```

Comment:
Probably unintended change?

Reply:
I need this for logging.
But we can just pull in package:logging into nextcloud as well

Reply:
We're already doing so

Reply:
my bad this is about the dynamite_runtime package. Should we pull it in?

Reply:
Yes let's do that